api/app/signals/apps/api/serializers/signal_context.py

```diff
@@ -98,3 +117,23 @@ def get_reporter(self, obj):
             'positive_count': satisfied_count,
             'negative_count': not_satisfied_count,
         }
+
+
+class SignalContextGeoSerializer(GeoFeatureModelSerializer):
+    location = GeometryField(source='location.geometrie')
+
+    class Meta:
+        model = Signal
+        id_field = False
+        geo_field = 'location'
+        fields = ['id', 'created_at']
+
+    def get_properties(self, instance, fields):
+        properties = super(SignalContextGeoSerializer, self).get_properties(instance, fields)
+        properties.update({
+            'status': {
+                'state': instance.status.state,
+                'state_display': instance.status.get_state_display(),
```

Comment:
We leave it up to the frontend to map the state values to something like pending/resolved (or open/closed). I like that as it makes this feature more general / future proof.